pkg/bmc/redfish_noop.go

```diff
@@ -0,0 +1,112 @@
+package bmc
+
+import (
+	"net/url"
+	"strings"
+)
+
+func init() {
+	schemes := []string{"http", "https"}
+	RegisterFactory("redfish-noop", newRedfishNoopAccessDetails, schemes)
+	RegisterFactory("ilo5-redfish-noop", newRedfishNoopAccessDetails, schemes)
```

Comment:
I'm not sure we need ilo5 support for it, ilo5 should work with boot mode (I hope).